### DISCUSSION TITLE
[FR]: Formula update field

### DISCUSSION THREAD

**Discussion by DannyHatcher-Git** [RafaelGB/obsidian-db-folder#522] [Q&A]
### Contact Details

danny@dannyhatcher.com

### Present your request

When I add a formula to a database, each row needs to be updated for the formula to work.

Is there a way to bulk update the formula column or do I need to do it all manually?
I would like the values to show when I add the formula.

### For which platform do you request this request??

Desktop

**Comment by RafaelGB** [comment]
depends of your formula and how the column is configured, but the formula is triggered when your table is opened

**Reply by DannyHatcher-Git** [reply]
The formula doesn't seem to remain sometimes.

**Reply by DannyHatcher-Git** [reply]
https://user-images.githubusercontent.com/78496825/196481941-ad790140-90f3-4d58-bb7d-c76ce2476b92.mp4

**Reply by RafaelGB** [reply]
could you share the formula itself?

If nothing is rendered is because an error was thrown

**Reply by DannyHatcher-Git** [reply]
${row.Date.toFormat("h")}

**Reply by DannyHatcher-Git** [reply]
It is the same for other date format changes
